src/components/NightMode/NightMode.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { NightMode } from './NightMode';
+
+export default {
+  title: 'Components/Night Mode',
```

Comment:
Once we have **dark** but also **light** I would recommend to use something not related to only one. So, what about `<ThemeMode>`?

Reply:
Thanks for that!